Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Logging/TransferLogSink.cs

```diff
@@ -4,62 +4,57 @@
 using Serilog.Core;
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/3b2b2fbd96e793fb58e0ddc3e006dc3f6fcb985c/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Logging/TransferLogSink.cs#L1-L3
You are not using these namespaces, so they can be removed.